dozer-api/src/rest/tests/routes.rs

```diff
@@ -257,3 +259,112 @@ async fn path_collision_test() {
     //assert the route matched something
     assert_ne!(res.status(), 404);
 }
+
+#[actix_web::test]
```

Comment:
Is it possible to test the `FromRequest` implementation without setting up the service?

Reply:
I'm just wondering if it's possible. If not testing with service is fine.